src/views/SystemIntake/index.tsx

```diff
@@ -61,135 +68,123 @@ export const SystemIntake = () => {
   };
 
   const dispatchSave = () => {
-    if (id && formikRef.current.dirty) {
-      dispatch(saveSystemIntake({ formData: formikRef.current.values, id }));
+    const currentRef = formikRef.current as FormikProps<SystemIntakeForm>;
```

Comment:
Is there a reason why you went with `as` syntax rather than:
`const currentRef: FormikProps<SystemIntakeForm> = formikRef.current;

Generally, I'd like to go with object restructuring here, but I'm not exactly sure what the Typescript syntax would look like. Maybe?

`const { current } : { current: FormikProps<SystemIntakeForm> } = formikRef;`